driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/MongoCollection.java

```diff
@@ -169,6 +169,7 @@
      * Counts the number of documents in the collection.
      *
      * <p>
+     * Note: For a fast count of the total documents in a collection see {@link #estimatedDocumentCount()}.
```

Comment:
nit: As this renders as HTML consider adding a `<br>` at the end of each line.

Currently this will publish as so:

>  Note: For a fast count of the total documents in a collection see estimatedDocumentCount(). Note: When migrating from count() to countDocuments() the following query operators must be replaced:

However, using a `<br>` at the end of the first line would render it as:

> Note: For a fast count of the total documents in a collection see estimatedDocumentCount().
> Note: When migrating from count() to countDocuments() the following query operators must be replaced:

Reply:
@rozza I have added `<br>` at the end of lines where there are multiple notes.

Reply:
LGTM!